docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Provision Kyma environment through Kyma Environment Broker
+type: Tutorials
+---
+
+This tutorial shows how to provision Kyma environment on Azure.
+
+## Prerequisites
+
+- Compass with:
+  * configured Runtime Provisioner and Azure provider [set up](../provisioner/08-02-provisioning-gardener.md)
+  * configured Kyma Environment Broker and chosen [overrides](03-06-runtime-overrides.md) set up
+
+## Steps
+
+1. Export these values as environment variables:
+```bash
+export BROKER_URL={KYMA_ENVIRONMENT_BROKER_URL}
+export INSTANCE_ID={INSTANCE_ID}
+export GLOBAL_ACCOUNT_ID={GLOBAL_ACCOUNT_ID}
+export NAME={RUNTIME_NAME}
+```
+
+> **NOTE:** INSTANCE_ID and NAME must be unique. It is recommended to use UUID as instance ID.
+
+2. Obtain Authorization credentials as described in [Authorization section](./03-05-authorization.md). Export this variable based on chosen authorization method:
+
+```bash
+export AUTHORIZATION_HEADER="Authorization: {Basic OR Bearer} $ENCODED_CREDENTIALS"
+```
+
+3. Make a call to the Kyma Environment Broker to create a runtime on Azure. List of possible request parameters is available [here](./03-01-service-description.md)
+
+```bash
+curl --request PUT "https://$BROKER_URL/v2/service_instances/$INSTANCE_ID?accepts_incomplete=true" \
+--header 'X-Broker-API-Version: 2.14' \
+--header 'Content-Type: application/json' \
+--header "$AUTHORIZATION_HEADER" \
+--header 'Content-Type: application/json' \
+--data-raw "{
+    \"service_id\": \"47c9dcbf-ff30-448e-ab36-d3bad66ba281\",
+    \"plan_id\": \"4deee563-e5ec-4731-b9b1-53b42d855f0c\",
+    \"context\": {
+        \"globalaccount_id\": \"$GLOBAL_ACCOUNT_ID\"
+    },
+    \"parameters\": {
+        \"name\": \"$NAME\"
```

Comment:
don't we also need now to specify the empty components list and region name?

Reply:
Only `name` parameter is required, the rest is optional (see [here](https://github.com/kyma-incubator/compass/blob/master/docs/kyma-environment-broker/03-01-service-description.md)) and will be set to defaults if not provided in request.